web/admin/src/Admin.js

```diff
@@ -87,7 +87,7 @@ export default class Admin extends PureComponent {
                     <div><label>Time per question: <input type="number" min="0" value={sessions[sessionId].secondsPerQuestion} onChange={this.onSecondsChange} /> seconds</label></div>
                     <div><label>Leaderboard up to <input type="number" min="1" max="1000" value={sessions[sessionId].leaderboardMax || 1000} onChange={this.onLeaderboardMaxChange} /> {th(sessions[sessionId].leaderboardMax)} place</label></div>
                     <div>
-                      <button onClick={() => this.addQuestion(this.state.sessionId)}>{this.returnHelpText()}</button>
+                      <button onClick={() => this.addQuestion(this.state.sessionId)} disabled={this.returnHelpText() ==="Complete Last Question"}>{this.returnHelpText()}</button>
```

Comment:
This will break when we add translations.  Display text should be based on state, and never the other way around.